operator/src/main/java/oracle/kubernetes/operator/IntrospectionStatus.java

```diff
@@ -149,7 +148,7 @@ Step createStatusUpdateSteps() {
 
       return Optional.ofNullable(getErrorMessage(pod))
             .map(m -> createFailureMessage(pod, m))
-            .map(f -> DomainStatusUpdater.createFailureRelatedSteps(ServerPod, f)).orElse(null);
```

Comment:
You can use a lambda here.

Reply:
I agree; but I followed what IntelliJ suggested after the method signature was changed to have only one parameter.